README.md

```diff
@@ -422,12 +422,47 @@ with at least one node that has persistent memory device(s). For development or
 testing, it is also possible to use a cluster that runs on QEMU virtual
 machines, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes) section below.
 
+- **Make sure that alpha feature gates CSINodeInfo, CSIDriverRegistry are enabled**
```

Comment:
Let's write a full sentence.

"Make sure that the alpha feature gates CSINodeInfo and CSIDriverRegistry are enabled."

Reply:
same here, these lines are actually bulleted headers, which dont have dots, as headers typically dont have

Reply:
I still think this reads poorly. Let's leave it as it is for now.

cc @MCamp859